src/main/java/teammates/sqllogic/core/CoursesLogic.java

```diff
@@ -121,6 +122,17 @@ public void deleteCourseCascade(String courseId) {
             return;
         }
 
+        usersLogic.deleteStudentsInCourseCascade(courseId);
+        List<FeedbackSession> feedbackSessions = fsLogic.getFeedbackSessionsForCourse(courseId);
+        feedbackSessions.forEach(feedbackSession -> {
+            fsLogic.deleteFeedbackSessionCascade(feedbackSession.getName(), courseId);
+        });
```

Comment:
Is it not possible to add `cascade` to the `OneToMany` annotation in `Course` and have this handled for us?